mpt/antlib/pom.xml

```diff
@@ -84,10 +84,10 @@ to the library requiring no extra coding.</description>
                         <configuration>
                             <tasks>
                                 <ant antfile="src/test/resources/build.xml" inheritAll="false" inheritRefs="false">
-                                    <property name="jar.name" location="${project.build.outputDirectory}" />
-                                    <property name="base.dir" location="${basedir}" />
-                                    <property name="test.skipped" value="${maven.test.skip}" />
-                                    <property name="test.classpath" refid="maven.test.classpath" />
+                                    <property location="${project.build.outputDirectory}" name="jar.name"></property>
+                                    <property location="${basedir}" name="base.dir"></property>
```

Comment:
when a fix has side effect, it's no more orthogonal.
but I agree that we can live with this one